app/filters/category_filter.rb

```diff
@@ -0,0 +1,17 @@
+class CategoryFilter < BasicFilter
+  def self.options
+    {
+      name: "Categories",
+      # Currently only filtering by top-level categories
+      filters: FilterOptionsBlueprint.render_as_hash(Category.roots)
+    }
```

Comment:
Serializing inside the filters doesn't feel quite right. Is it possible to have `options` return a relation as the value of `filters` and let the controller handle serialization?
```suggestion
    {
      name: "Categories",
      # Currently only filtering by top-level categories
      filters: Category.roots
    }
```

This might require introducing another blueprint at the level of a filter that then has an `association :filters`.

Reply:
The only with your suggestion is that the Ask/Offer distinction is one I'm doing manually:

https://github.com/rubyforgood/mutual-aid/blob/39a0d099aaa860edf396a000e86ee8bdb8397237/app/filters/contribution_type_filter.rb#L1-L7

Reply:
Ah yes, i see the challenge. I had a go at extracting the serialization out in 8b8c8a94b5dd1e50f256e12abbd3071550ebed61, curious your thoughts?
In any case, i don't think this question should hold up this PR.